workflow_deeplabcut/paths.py

```diff
@@ -1,11 +1,12 @@
 import datajoint as dj
+from collections import abc
 
 
 def get_dlc_root_data_dir():
     dlc_root_dirs = dj.config.get("custom", {}).get("dlc_root_data_dir")
     if not dlc_root_dirs:
         return None
-    elif not isinstance(dlc_root_dirs, list):
+    elif not isinstance(dlc_root_dirs, abc.Sequence):
```

Comment:
Is there a particular reason for using `abc.Sequence`? It's very uncommon and introduces extra line at the top compared to `list`

Reply:
This is something I saw Dimitri adjust elsewhere. I think the idea is that `Sequence` is a superordinate category, so this would do a better job catching edge cases

Reply:
Looks like `'', (), [], {}, set(), range(0)` count as `abc.Sequence`

Reply:
I think empty sequences would get caught by the previous line `if not dlc_root_dirs` before getting the this evaluation. I think the only place the result gets used is as passed to `find_full_path` which passes the content to a for loop. I think the for loop will work for any of the above types so long as they resolve to a valid path